contrib/custom-image-kubelet/README.md

```diff
@@ -24,7 +24,7 @@ The remaining RKE2 components include:
 Requirements
 ----
 
-1. RKE2 v1.18.12-beta1+rke2r2 or newer (with image/kubelet override support)
+1. RKE2 v1.18.13-beta1+rke2r2 or newer (with image/kubelet override support)
```

Comment:
I know this was probably just caught by sed, but we might as well bump to full GA release
```suggestion
1. RKE2 v1.18.13+rke2r1 or newer (with image/kubelet override support)
```